### ISSUE TITLE
MS SQL Server #temp tables invalid object name

### ISSUE THREAD

**Issue by IsaaacD** [dbfit/dbfit#430]
![image](https://cloud.githubusercontent.com/assets/5303125/10048124/7cc0cf0c-61df-11e5-9e79-534353bd0345.png)

Creating a temporary table and trying to use it within the same test throws an exception as seen above.

**Comment by Auguronomics** [comment]
There is a workaround : 
Try to use the Execute DDL statement it'll work

!|Execute Ddl|Create table #tempResults(ClientID varchar(10), NetworkAccount varchar(100), AccountType varchar(10) )|

!|Execute|Insert Into #tempResults VALUES ('value1','value2','value3')|

!|query| Select * From #tempResults|
|ClientID|NetworkAccount|AccountType|
|value1|value2|value3|

**Comment by ghall1t** [comment]
Hey I tried using the **Execute Ddl** workaround and it didn't work for me, output attached.

[Execute_Ddl_output.txt](https://github.com/dbfit/dbfit/files/3206457/Execute_Ddl_output.txt)

**Comment by MMatten** [comment]
It looks like you’re using the old FitSharp-based version of DbFit rather than the newer Java-based version. The java version includes the `Execute Ddl` fixture.

**Comment by ghall1t** [comment]
Thanks @MMatten I think I see at least part of the problem... 

I'm using version 3.2.0 which I got from [http://dbfit.github.io/dbfit/](http://dbfit.github.io/dbfit/) earlier this year. It _does_ have a ../fitsharp/ sub-directory, and I've been setting:

> !define TEST_RUNNER {fitsharp\Runner.exe} 

As I kept getting a variety of complaints without that and didn't find a solution at the time (e.g. _java.io.IOException: Cannot run program "fit.FitServer": CreateProcess error=2, The system cannot find the file specified_) 

Your mention of fitsharp sent me off looking for tips again and I suspect that I may have missed a requirement to separately grab [FitLibrary](https://sourceforge.net/projects/fitlibrary/) so I've grabbed that and am going to give it a shot, does it sound like I'm on the right track to you?

**Comment by MMatten** [comment]
To use the java test runner just use the default test runner (remove the !define TEST_RUNNER line and any pattern definition too).

**Comment by ghall1t** [comment]
Yes I see that should work according to documentation, however if I remove definition of those elements in an effort to let the defaults take over I am presented with _Could not find fixture_ exceptions. See attached image ![2019-05-24_Could_not_find_fixture_ConnectUsingFile](https://user-images.githubusercontent.com/48008251/58301210-19486300-7e18-11e9-8a8e-7cf5562c7f91.png) and 
[2019-05-24-content.txt](https://github.com/dbfit/dbfit/files/3214977/2019-05-24-content.txt) for the full content of the page.

Though this issue is the root cause of me heading down this path, it's starting to seem like a different issue really. Happy to split it out if I should.
